src/components/atoms/Link/Link.tsx

```diff
@@ -1,22 +1,23 @@
 import React from 'react';
 import styled from 'styled-components';
-import { colors } from '../../../constants/colors';
+import { colors, TLinkHexColors } from '../../../constants/colors';
 import { typography } from '../../../constants/typography';
 
 export interface ILinkProps
   extends React.AnchorHTMLAttributes<HTMLAnchorElement>,
     React.RefAttributes<HTMLAnchorElement> {
   target?: '_blank';
-  color?: any; // see issue #139 for more context on why the explicit `any` here.
+  color?: TLinkHexColors;
```

Comment:
you can probably make it required now here since there is no default anymore and the fallback is provided in the wrapper

Reply:
`ILinkProps` is used to type both `<Link />` and `<SLink />`, if we make it required, we would need to create a new `interface` to **type `<SLink color />` only required**: seems a bit of an overkill for the little benefit it brings?  💆🏻‍♂️